### PR TITLE
create tidb cluster on ack with cr

### LINKED ISSUE #1914: Deploy TiDB Cluster with CR via TiDB Operator v1.1 on Aliyun
## Feature Request

**Is your feature request related to a problem? Please describe:**

Deploy TiDB Cluster with CR via TiDB Operator v1.1 on Aliyun
**Describe the feature you'd like:**

- Terraform scripts only deploy infrastructure.
- Provide CR example for CR supported components
- Update doc in https://github.com/pingcap/docs-tidb-operator/

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What problem does this PR solve? 
Fix #1914 
### What is changed and how does it work?

- Add a variable to the terraform module to not install tidb cluster.
- Add CR example.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

    -  Deploy ack with terraform and then install tidb cluster and tidb monitor with the CRs.
    -  Upgrade tidb cluster.
    -  Update tidb cluster configuration.
    -  Scale out the tidb cluster.
    -  Deploy multiple tidb clusters.

Code changes

 - Has Terraform scripts change

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
Support deploying TiDB Cluster with TidbCluster and TidbMonitor CRs via Terraform on ACK.
```

### CODE REVIEW

File: deploy/aliyun/variables.tf [hunk 1]
```diff
@@ -112,3 +112,13 @@ variable "vpc_cidr" {
   description = "VPC cidr_block, options: [192.168.0.0.0/16, 172.16.0.0/16, 10.0.0.0/8], cannot collidate with kubernetes service cidr and pod cidr. Cannot change once the vpc created."
   default     = "192.168.0.0/16"
 }
+
+variable "create" {
```
Comment (cofyc) [inline deploy/aliyun/variables.tf]: `create-tidb-cluster`, make it clear that this controls whether to create a tidb cluster?

File: deploy/aliyun/manifests/db-monitor.yaml.example [hunk 2]
```diff
@@ -0,0 +1,86 @@
+apiVersion: pingcap.com/v1alpha1
```
Comment (cofyc) [inline deploy/aliyun/manifests/db-monitor.yaml.example]: what are example files for?

File: deploy/aliyun/variables.tf (hunk 1 above)
Comment (DanielZhangQD) [reply deploy/aliyun/variables.tf]: OK.

File: deploy/aliyun/manifests/db-monitor.yaml.example (hunk 2 above)
Comment (DanielZhangQD) [reply deploy/aliyun/manifests/db-monitor.yaml.example]: These the CRs that including the special configuration on ACK, e.g. LB config and storage class, I will refer to these examples in the doc.

File: deploy/aliyun/variables.tf (hunk 1 above)
Comment (DanielZhangQD) [reply deploy/aliyun/variables.tf]: done.

**Comment by DanielZhangQD** [comment]
@cofyc comments addressed, PTAL again.

**Pull request merged**